src/style.css

```
@@ -0,0 +1,39 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.container {
+  width: fit-content;
+}
+
+.stars {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background-image: url("images/star-active.svg");
+  margin-right: 4px;
+  border-radius: 0.5px;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars.stars--0 > div,
+.stars.stars--1 > div:nth-child(n + 2),
```

you can use for example to avoid style tags
```suggestion
.stars--1 .stars__star:nth-child(-n + 1),
.stars--2 .stars__star:nth-child(-n + 2),
```